ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -142,6 +142,40 @@ mac_address_table:
       underlay_l2_multicast_group_ipv4_pool: < IPv4_address/Mask >
       underlay_l2_multicast_group_ipv4_pool_offset: < int >
 
+    # Enable L3 Multicast for all SVIs and l3vlans within Tenant | Optional
+    # - In the evpn-l3ls design type, this enables L3 EVPN Multicast (aka OISM)
+    # - Multicast group binding for VRF is created only for Multicast traffic. BULL traffic will use ingress-replication
+    # - Configures binding between VXLAN, VLAN, and multicast group ipv4 address using the following formula:
+    #   < l3_multicast.evpn_underlay_l3_multicast_group_ipv4_pool > + < vrf_vni - 1 > + < l3_multicast.evpn_underlay_l3_multicast_group_ipv4_pool_offset >.
+    # - The recommendation is to assign a /20 block within the 232.0.0.0/8 Source-Specific Multicast range.
+    # - If enabled on an SVI using the anycast default gateway feature, a diagnostic loopback (see below) must be configured to source IGMP traffic.
+    # - Enables "evpn multicast" on the router bgp VRF.
+    # - When enabled on an SVI:
+    #     - If switch is part of an MLAG pair, enables "pim ipv4 sparse-mode" on the SVI.
+    #     - If switch is standalone or A-A MH, enables "ip igmp" on the SVI.
+    #     - If "ip address virtual" is configured, enables "pim ipv4 local-interface" and uses the diagnostic Loopback defined in the VRF
+    l3_multicast:
+      enabled: < true | false | default false >
+      evpn_underlay_l3_multicast_group_ipv4_pool: < IPv4_address/Mask > # Required
+      evpn_underlay_l3_multicast_group_ipv4_pool_offset: < int > # Optional
+      node_settings: # Optional
+        # For each group of nodes, allow configuration of EVPN PEG features | Optional
+        # and allow configuration of specific RPs and associated multicast groups
+        # Designed to be used in conjunction with enabling PIM on l3_interfaces to make the device a PEG (see below)
+        # An individual node should be in one node group only
```

Comment:
is the code checking for this? (I guess the `break` will prevent the code to be executed for both)

Reply:
Correct; the break means that only one set of settings will apply.